src/mongo/base/DriverBase.class.php

```diff
@@ -203,7 +203,7 @@ protected function fetchGraph($query, $type, $collection=null,$includeProperties
                     if ($type==MONGO_VIEW && array_key_exists(_EXPIRES,$result['value']))
                     {
                         // if expires < current date, regenerate view..
-                        $currentDate = new UTCDateTime(floor(microtime(true))*1000);
+                        $currentDate = new UTCDateTime(floor(microtime(true) * 1000));
```

Comment:
Can we move this into a util function of some description we are doing this everywhere, lets be DRY